src/utils/assets/elements/touchable/pipe.py

```diff
@@ -1,11 +1,11 @@
-from pygame import image, transform
+from src.utils.assets.players.create_images import create_image
+from src.utils.constants import GENERAL_SIZE
+from src.utils.directories import PIPE_BODY_DIR, PIPE_HEAD_DIR
 
-from src.utils.directories import PIPE_DIR
 
-BIG_PIPE = transform.scale(image.load(PIPE_DIR + "big-pipe.png"), (100, 200))
-MEDIUM_PIPE = transform.scale(
-    image.load(PIPE_DIR + "medium-pipe.png"), (100, 150)
-)
-SMALL_PIPE = transform.scale(
-    image.load(PIPE_DIR + "small-pipe.png"), (100, 100)
-)
+PIPE_BODY_STYLE_1 = create_image(PIPE_BODY_DIR + "style_1.png", GENERAL_SIZE)
```

Comment:
Why u use a general size for each image, a cloud maybe could have a different width than a pipe, the size doesn't be defined for us, the image should defined for the asset instead

Reply:
I use a GENERAL SIZE for each image, because most of the images have a general size, now for the issue of different sizes for each block there are specific constants for each certain size, also more sizes will be added, but in another PR that is focused for the ASSETS as the background, clouds, etc..

There are GENERAL constants for the large number of existing elements and SPECIFIC constants for certain elements.

Maybe a way to fix this would be to create a wiki where each constant is specified for each element.